app/src/main/java/com/cryart/sabbathschool/ui/about/SSAboutViewModel.kt

```diff
@@ -30,6 +31,8 @@ class SSAboutViewModel(private val context: Context) {
 
     val colorPrimary: Int get() = context.colorPrimary
 
+    val versionInfo: String get() = "${context.getString(R.string.ss_settings_version)}: ${BuildConfig.VERSION_NAME}"
```

Comment:
Not sure this is right way for RTL languages. It's better to create a formatted record in `string.xml`, like `<string name="ss_settings_version">Version %1$s</string>`

Reply:
updated and fixed ;-)